### DISCUSSION TITLE
Add more repository "managers"

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#2227] [Ideas]
Adding 1 or 2 users who can help "manage" the repository might be beneficial. They would have specific permissions like labeling PRs/issues, closing issues/PRs, and managing the Discussions. This would help bring the organization back to the issues and discussions. I understand that it is super hard to manage a massive repository by yourself, especially with other (more important) things going on in life. For example, #2202 could be closed.

I'm unsure how, but I believe adding "Collaborators" to the repo with special permissions might be possible. They can sort of be "community managers" but can't merge pull requests, delete branches, or change the code base. 

You could choose people who are active in the community and/or understand customtkinter, Python, and/or tkinter.

**Comment by Akascape** [comment]
@TomSchimansky
